resqpy/rq_import/_GridFromCp.py

```diff
@@ -0,0 +1,555 @@
+"""_GridFromCp.py: Class to generate a RESQML grid object from an input corner point array, and other optional inputs."""
```

Comment:
I'd like us to change the name of this file – we are trying to avoid CamelCase in the python file names. I see a couple of options:

- merge the code into _grid_from_cp.py
- give this file a different name such as: _grid_from_cp_cls.py